### DISCUSSION TITLE
Maven Dependency Not Working?

### DISCUSSION THREAD

**Discussion by JonahTheEngi1** [Minestom/Minestom#2124] [Help]
Hello there, I'm wanting to make an extension with Minestom however whilst adding it as a dependency in my pom.xml i get

`Could not find artifact com.github.Minestom:Minestom:pom:Ta in jitpack.io (https://jitpack.io)`

Here is my pom.xml

`<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>net.atlaspvp</groupId>
    <artifactId>hubextension</artifactId>
    <version>1.0</version>
    <packaging>jar</packaging>

    <name>Hubextension</name>

    <properties>
        <java.version>1.8</java.version>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    </properties>

    <build>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.8.1</version>
                <configuration>
                    <source>${java.version}</source>
                    <target>${java.version}</target>
                </configuration>
            </plugin>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-shade-plugin</artifactId>
                <version>3.2.4</version>
                <executions>
                    <execution>
                        <phase>package</phase>
                        <goals>
                            <goal>shade</goal>
                        </goals>
                        <configuration>
                            <createDependencyReducedPom>false</createDependencyReducedPom>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
        </plugins>
        <resources>
            <resource>
                <directory>src/main/resources</directory>
                <filtering>true</filtering>
            </resource>
        </resources>
    </build>

    <repositories>
        <repository>
            <id>jitpack.io</id>
            <url>https://jitpack.io</url>
        </repository>
    </repositories>

    <dependencies>
        <dependency>
            <groupId>com.github.Minestom</groupId>
            <artifactId>Minestom</artifactId>
            <version>Tag</version>
        </dependency>
    </dependencies>
</project>`

**Comment by JonahTheEngi1** [comment]
Im dumb please ignore

**Reply by AchyMake** [reply]
for a new guy i would like to know :c
